src/app/src/main/kotlin/org/imperial/mrc/hint/controllers/ModelRunController.kt

```diff
@@ -0,0 +1,19 @@
+package org.imperial.mrc.hint.controllers
+
+import org.imperial.mrc.hint.APIClient
+import org.imperial.mrc.hint.FileManager
+import org.imperial.mrc.hint.models.ModelRunParameters
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.*
+
+@RestController
+@RequestMapping("/model")
+class ModelRunController(val fileManager: FileManager, val apiClient: APIClient) {
+
+    @PostMapping("/run/")
+    @ResponseBody
+    fun uploadSurvey(@RequestBody params: ModelRunParameters): ResponseEntity<String> {
```

Comment:
This function should be `runModel` or something?

Reply:
There is no test that calls this itself either? That will come when hook up the UI to this?

Reply:
Oops, good catch! There is an integration test that calls this endpoint, I thought it was overkill to have a controller test as well given how simple it is...